src/plugins/wizard/public/application/components/data_tab/secondary_panel.tsx

```diff
@@ -39,9 +41,21 @@ export function SecondaryPanel() {
       indexPattern && draftAgg && aggService.createAggConfigs(indexPattern, [cloneDeep(draftAgg)])
     );
   }, [draftAgg, aggService, indexPattern]);
-
   const aggConfig = aggConfigs?.aggs[0];
 
+  const metricAggs = useMemo(
+    () =>
+      indexPattern
+        ? aggService.createAggConfigs(
+            indexPattern,
+            cloneDeep(
+              aggConfigParams.filter((aggConfigParam) => aggConfigParam.schema === 'metric')
+            )
+          ).aggs
+        : [],
+    [aggConfigParams, aggService, indexPattern]
+  );
```

Comment:
This seems like a bit of a hack (to have metric visualization-specific logic here instead of in `/visualizations/metric`), but I may be misunderstanding what `aggConfigParam.schema === 'metric'` means in this context. A comment may be helpful to clarify either way.

Reply:
Good call, will add a comment in a followup PR, but yes metric aggregations are different from the metric visualization. Aggregations are classiied into 2 main categories, metric and bucket aggregations. The bar, line and area chart visualizations need to know if any metric aggregations are performed to correctly calculate the auto bounds of a histogram visualization.

Reply:
ah, got it, that makes sense